fendermint/app/options/src/genesis.rs

```diff
@@ -147,6 +147,28 @@ pub enum GenesisIpcCommands {
     Gateway(GenesisIpcGatewayArgs),
     /// Fetch the genesis parameters of a subnet from the parent.
     FromParent(Box<GenesisFromParentArgs>),
+    /// Seal the genesis state from the genesis parameter file
+    SealState(SealGenesisArgs),
+}
+
+#[derive(Args, Debug, Clone)]
+pub struct SealGenesisArgs {
+    /// The built in actors bundle path
+    #[arg(long, short)]
+    pub builtin_actors_path: PathBuf,
+
+    /// The custom actors bundle path
+    #[arg(long, short)]
+    pub custom_actors_path: PathBuf,
+
+    /// The ipc artifacts output path. If you are using ipc-monorepo, it should be the `out` folder
+    /// of `make build`
+    #[arg(long, short)]
+    pub ipc_artifacts_path: PathBuf,
```

Comment:
I wonder if we can embed these artifacts in the build, and load them from the target directory instead?

Reply:
@cryptoAtWill, any thoughts here?

Reply:
oops, missed this. I think we have discussed this before, if I recall correctly, it's because `ipc_contracts` can be updated and compile it will result in a different artifacts build. Using a path gives more flexibility on which artifacts should be imported.